recipes/cinder.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Cookbook Name:: osl-openstack
+# Recipe:: cinder
+#
+# Copyright (C) 2015 Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# this is required because of the fedora deps. Will be fixed once its moved into
+# a _common recipe.
+iscsi_hosts = ['127.0.0.1']
+iscsi_hosts << node['osl-openstack']['cinder']['iscsi_ips'] unless
```

Comment:
is `iscsi_ips` suposed to be an Array? If so, you're going to get nested arrays:

```
irb(main):001:0> a = [1,2,3,4]
=> [1, 2, 3, 4]
irb(main):002:0> b = [5,6,7,8]
=> [5, 6, 7, 8]
irb(main):003:0> a << b
=> [1, 2, 3, 4, [5, 6, 7, 8]]
```